.github/workflows/backport.yml

```diff
@@ -9,6 +9,7 @@ jobs:
   backport:
     name: Backport PR
     runs-on: ubuntu-latest
+    if: github.event.pull_request.merged && contains(github.event.pull_request.labels.*.name, 'backport-to-')
```

Comment:
So was the example yml file no good because "labeled" in the `type` causes the action to run when the label is added, not just merging a PR with the label?  That's screwy, but oh well.

Reply:
yes, here https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#label it explains it, I think we don't want that behavior. 
But in any case, we cannot avoid this workflow got triggerred every time there is a pr got merged